tools/psa/build_tfm.py

```diff
@@ -419,6 +420,37 @@ def _copy_binaries(source, destination, toolchain, target):
                                             relpath(output_dir, ROOT)))
             shutil.copy2(tfm_secure_bin, output_dir)
 
+def _copy_tfm_ns_files(source):
+    """
+    Copy TF-M NS API files into Mbed OS
+    :param source: Source directory containing TF-M NS API files
+    """
+    with open(join(dirname(__file__), "tfm_ns_import.json")) as ns_import:
+        json_data = json.load(ns_import)
+        data_files = json_data["files"]
+        data_folders = json_data["folders"]
+        logger.info("Copying NS API source from TF-M to Mbed OS")
+        for f in data_files:
+            src_file = join(source, f["src_file"])
+            dst_file = join(ROOT, f["dst_file"])
+            if not isdir(dirname(dst_file)):
+                os.makedirs(dirname(dst_file))
+            try:
+                shutil.copy2(src_file, dst_file)
+            except FileNotFoundError:
+                # Wordaround to handle app/os_wrapper_cmsis_rtos_v2.c located
+                # in the root directory of TF-M
```

Comment:
Could you explain this comment more to explain what the workaround is and why it is needed? It's not clear why this file can't be handled in a manner consistent with the other files.

Why does a file sourced from `install/export/` get treated differently from a file sourced from `app/`?

Reply:
Updated the comment with more details.